BDArmory/Weapons/Missiles/MultiMissileLauncher.cs

```diff
@@ -151,6 +155,7 @@ IEnumerator DelayedStart()
                 if (!isClusterMissile) //Clustermissiles replace/generate MMR on launch, other missiles should have it in the .cfg
                     Debug.LogError($"[BDArmory.MultiMissileLauncher] no ModuleMissileRearm on {part.name}. Please fix your .cfg");
             }
+            //missileSpawner.ammoCount = launchTransforms.Length; //FIXME: remember to remove this for standard builds
```

Comment:
Should this line be fully removed or do you want to keep it around as an option/toggle/hidden toggle?

Reply:
Hmm. I can see a use for it for people running competitions where clamping multimissilelaunchers to a single unit of ordinance would be useful, so keeping it as a hidden toggle would work.

Reply:
It disables reloading on reloadable missile launchers that are carrying additional ordinance for reloading, so you only get the single launch/salvo, and once the launcher is dry, that's it. Any additional ammo carried by the launcher is removed on launch/spawn.
If Infinite Ordinance was also enabled, for whatever reason, it would override this and continue to provide infinite missiles, since that doesn't care what the initial ammo count of the launcher is.